internal/client/client.go

```diff
@@ -88,11 +88,11 @@ func (c *Config) Validate() error {
 	return nil
 }
 
-type ShutdownHandler func(ctx context.Context)
+type shutdownHandler []func(context.Context)
```

Comment:
Why do we need this change?

Reply:
Resolved, [0042502](https://github.com/hazelcast/hazelcast-go-client/pull/772/commits/0042502d165af4bbfd57b99474f816dd147c4599).